test/functional/ui/highlight_spec.lua

```diff
@@ -303,3 +303,60 @@ describe('Default highlight groups', function()
     ]], {[1] = {foreground = Screen.colors.Red, background = Screen.colors.Green}})
   end)
 end)
+
+describe('New special highlight color attribute', function()
+  local screen
+
+  before_each(function()
+    clear()
+    screen = Screen.new(25,10)
+    screen:attach()
+    screen:set_default_attr_ignore({
+      {bold=true, foreground=Screen.colors.Blue}
+    })
+  end)
+
+  it('used for undercurls', function()
```

Comment:
This test did not fail after commenting out these lines:

https://github.com/neovim/neovim/pull/4633/commits/b1273e1c082233082e4d9f7b247b241057e80a64#diff-cff0db2d9810ddbe3e259dacfece5acfR335

And this entire block: https://github.com/neovim/neovim/pull/4633/commits/b1273e1c082233082e4d9f7b247b241057e80a64#diff-cbd63c814d2878576ddab95ccb80b655R6460

What's the test actually testing?